### ISSUE TITLE
share my results

### ISSUE THREAD

**Issue by JiyueWang** [luanyunteng/pytorch-be-your-own-teacher#3]
04-24-20 06:48:Epoch [199] learning rate = 0.00010000000000000003
04-24-20 06:48:Epoch: [199]	Iter: [0]	Time 3.853 (3.853)	Data 3.780 (3.780)	Loss 9.838 (9.838)	Prec@1 100.000 (100.000)	
04-24-20 06:48:Epoch: [199]	Iter: [100]	Time 0.142 (0.174)	Data 0.000 (0.038)	Loss 9.809 (9.795)	Prec@1 100.000 (99.985)	
04-24-20 06:49:Epoch: [199]	Iter: [200]	Time 0.142 (0.158)	Data 0.000 (0.020)	Loss 9.676 (9.779)	Prec@1 100.000 (99.977)	
04-24-20 06:49:Epoch: [199]	Iter: [300]	Time 0.145 (0.153)	Data 0.000 (0.013)	Loss 9.753 (9.789)	Prec@1 100.000 (99.958)	
04-24-20 06:49:Loss 0.893	Prec@1 79.170	Middle1@1 72.170	Middle2@1 76.250	Middle3@1 78.320	
04-24-20 06:49: * Prec@1 79.170
best:  tensor(79.4100, device='cuda:0')

I change the bottleneck layer to this:
def branchBottleNeck(channel_in, channel_out, kernel_size):
    # middle_channel = channel_out//4
    middle_channel = channel_out//2

    return nn.Sequential(
        nn.Conv2d(channel_in, middle_channel, kernel_size=1, stride=1),
        nn.BatchNorm2d(middle_channel),
        nn.ReLU(),
        
        nn.Conv2d(middle_channel, middle_channel, kernel_size=kernel_size, stride=1),
        nn.BatchNorm2d(middle_channel),
        nn.ReLU(),
        nn.Conv2d(middle_channel, middle_channel, kernel_size=kernel_size, stride=kernel_size),
        nn.BatchNorm2d(middle_channel),
        nn.ReLU(),
        
        nn.Conv2d(middle_channel, channel_out, kernel_size=1, stride=1),
        nn.BatchNorm2d(channel_out),
        nn.ReLU(),
        )

**Comment by JiyueWang** [comment]
In my opinion, the branch bottleneck should be stronger so that the basic blocks do not need to learn hight level feature desperately. This structure is my first try, a deeper structure may help further.

**Comment by JiyueWang** [comment]
Can you share more of your results with different settings?
